bin/happo-ci

```diff
@@ -18,6 +18,8 @@ if [ -n "$PREVIOUS_SHA" ]; then
 fi
 CURRENT_SHA="$(${HAPPO_GIT_COMMAND} rev-parse "$CURRENT_SHA")"
 
+HAPPO_FALLBACK_SHAS=${HAPPO_FALLBACK_SHAS:-$(${HAPPO_GIT_COMMAND} log --format=%H --first-parent --max-count=50 "$PREVIOUS_SHA"^)}
```

Comment:
It might be a good idea to make the max count configurable via an env var

Reply:
Will do, sounds like a good idea.